tsconfig.json

```diff
@@ -17,7 +17,8 @@
         "target": "es2015",
         "module": "es2020",
         "paths": {
-            "@components/*": ["app/components/*"]
+            "@components/*": ["app/components/*"],
```

Comment:
Let's keep the paths like this.
It makes the imports look clean.

Reply:
Yep! (Didn't know why I/VS Code deleted it...)